test/services/notifications/notificationSenderTest.js

```diff
@@ -0,0 +1,216 @@
+const createNotificationSender = require('../../../server/services/notifications/notificationSender')
+const templates = require('../../../server/services/config/notificationTemplates')
+
+describe('notificationSender', () => {
+  let service
+  let notifyClient
+  let audit
+  let config
+
+  beforeEach(() => {
+    config = {
+      notifications: {
+        notifyKey: 'dummy-key',
+        clearingOfficeEmail: 'HDC.ClearingOffice@justice.gov.uk',
+        clearingOfficeEmailEnabled: 'YES',
+        activeNotificationTypes: [
+          'CA_RETURN',
+          'CA_DECISION',
+          'RO_NEW',
+          'RO_TWO_DAYS',
+          'RO_DUE',
+          'RO_OVERDUE',
+          'DM_NEW',
+          'DM_TO_CA_RETURN',
+        ],
+      },
+      domain: 'http://localhost:3000',
+    }
+
+    notifyClient = {
+      sendEmail: sinon.stub().resolves({}),
+    }
+    audit = {
+      record: sinon.stub().resolves({}),
+    }
+
+    service = createNotificationSender(notifyClient, audit, config)
+  })
+
+  describe('notify', () => {
+    it('should do nothing if template ID not in active template list', async () => {
+      const notifications = [{ email: ['email@email.com'] }]
+      await service.notify({
+        sendingUserName: 'username',
+        bookingId: 123,
+        notifications,
+      })
+      expect(notifyClient.sendEmail).not.to.be.calledOnce()
+    })
```

Comment:
Would the test pass if sendEmail was called twice? Perhaps 
`expect(notifyClient.sendEmail).not.to.be.called()`
 would be better.

Reply:
probably copied this from somewhere - will fix in a follow up,